client/state/selectors/is-requesting-timezones.js

```diff
@@ -0,0 +1,10 @@
+
+/**
+ * Returns true if we are requesting the timezones
+ *
+ * @param  {Object}  state  Global state tree
+ * @return {Boolean} - Whether timezones is being requested
+ */
+export default function isRequestingTimezones( state ) {
+	return state.timezones.requesting;
```

Comment:
using something like `get` (a safe-defaulting getter) will prevent us from introducing type errors

this code invites it when `state.timezones` is `undefined`